contracts/contracts/ERC677/mock/token677.cairo

```diff
@@ -0,0 +1,33 @@
+%lang starknet
```

Comment:
Why would you need this specific mock contract that looks exactly like linkToken.Cairo already defined. Why not just use that one?

Reply:
Done, the file has been deleted